### PR TITLE
feat: user dashboard page ui created

### PR DESCRIPTION
**Pull request by PeterMichael28**

### CODE REVIEW

File: Client/src/components/NavBar.jsx [hunk 1]
```diff
@@ -23,41 +22,41 @@ const NavBar = () => {
...
-  //       .catch((error) => {
-  //         console.log("error", error);
-  //       });
-  //   }
-  // };
-  // const handleClick = async () => {
-  //   await logOut();
-  //   navigate("/Home");
-  // };
+  const logOut = async () => {
```
Comment (oluwatobiiloba) [inline Client/src/components/NavBar.jsx:25]: Kindly rework this logout function to use the serverAPI file and toast just like we do with login.

File: Client/src/components/NavBar.jsx [hunk 2]
```diff
@@ -23,41 +22,41 @@ const NavBar = () => {
...
+    let myHeaders = new Headers();
+    myHeaders.append("Content-Type", "application/json");
```
Comment (oluwatobiiloba) [inline Client/src/components/NavBar.jsx:27]: Same as this too, all fetch calls should be replaced with serverApi calls.

Comment (oluwatobiiloba) [reply Client/src/components/NavBar.jsx:27]: See this a s a reference.


`const login = async () => {
    let raw = JSON.stringify({
      password: user.Password,
      username: user.UserName,
    });
    const abortController = new AbortController();
    try {
      const response = await serverApi.post(
          '/authenticate/login',
          raw,
          {
            signal: abortController.signal,
            headers: {
              'content-type': 'application/json',
            }
          }
      )
       console.log(response);

       if (response.status === 200) {
         navigate("/dashboard");
         const responseData = response.data;
         const userInfo = {
           ...responseData.data,

    
          
            
    

          
          Expand Down
    
    
  
        };
        setUserProfile(userInfo.user);
        setIsLoggedIn(true);
        sessionStorage.setItem("userData", JSON.stringify(userInfo.user));
        sessionStorage.setItem("user_token", response.data.token);
        //localStorage.setItem("user", JSON.stringify(userInfo));
        if (userInfo.user.role === "admin") {
          setDashboardToggle({
            displayToggle: true,
            toggleValue: "User Dashboard",
          });
        }
      }
    
    } catch (error) {
      setUser({ error: error.response?.data?.message || "Failed, Please contact admin", UserName: "", Password: "" });
    }
    
  }`

File: Client/src/components/NavBar.jsx (hunk 1 above)
Comment (PeterMichael28) [reply Client/src/components/NavBar.jsx:25]: Ooh okay . I didn't even touch it . I met it that way and I didn't even bother to check it . But I will look into it

**Pull request merged**